ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_creation.py

```diff
@@ -600,6 +600,33 @@ def test_jax_identity(
     )
 
 
+# integer
+@handle_frontend_test(
+    fn_tree="jax.numpy.int64",
```

Comment:
If the function you've implemented is `jax.numpy.integer`, the `fn_tree` should be `jax.numpy.integer` otherwise it's trying to test a different function

Reply:
I did try `fn_tree="jax.numpy.integer"` but it gave me error on running pytest specifically for my created test using `sudo python3 -m pytest ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_creation.py::test_jax_integer` after I changed it to `jax.numpy.int64` it did not give me any error. 

I took reference of how the [test for double](https://github.com/unifyai/ivy/blob/c202a2a3bc9869b15fb38e00bf004b0b26754184/ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_creation.py#L312) was written and checked docs of [jax.numpy](https://jax.readthedocs.io/en/latest/jax.numpy.html).

I think the other fail checks here apart from semantics and lint ones were already there since I tried running `sudo python3 -m pytest ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_creation.py` on an unmodified ivy repo and still got many errors.

I think I might have to create a new PR since I did not follow the commit standards from start. Any thing I should try while I am creating another PR after closing this one?

Reply:
`TypeError: generic.__init__() got an unexpected keyword argument 'x'` is this a familiar error?

Reply:
It looks like a function somewhere is being called with `x` as a keyword argument when it should be a positional argument. If you look at the stack trace for that error, it should tell you which function that is, and then you can hopefully use that to fix it